ci/test

```diff
@@ -176,7 +176,35 @@ _build_conjur() (
   version_tag
 )
 
+# TODO: Possibly move this into bash-lib: will be useful elsewhere
+docker_diagnostics() {
+  # Read the running containers names into an array.
+  #
+  # Note on bash trickiness: The backslash is required in \$NF because we want
+  # $NF to be passed literally to awk -- so awk can interpret it -- whereas we
+  # want the shell to interpret and expand ${COMPOSE_PROJECT_NAME}, so awk sees
+  # its _value_ in the regex.
+  #
+  # Docker Note: container name is always the last field.  Hence $NF gets it.
+  local cont_names
+  readarray -t cont_names < <(docker ps --all | \
+    awk "/${COMPOSE_PROJECT_NAME}/{print \$NF}")
+
+  # Print docker inspect info
+  for name in "${cont_names[@]}"; do
+    echo "CONTAINER: ${name}"
+    docker inspect "$name"
+  done
+
+  # Print container logs
+  for name in "${cont_names[@]}"; do
+    echo "CONTAINER: ${name}"
+    docker logs "$name"
+  done
+}
```

Comment:
This seems overly complicated. What about:
```
for cid in $(docker-compose ps -q); do
  name=$(docker inspect --format="{{.Name}}" "$cid")
  echo "CONTAINER: $name"
  docker inspect "$cid"

  echo "CONTAINER: $name"
  docker logs "${name#/}"
done
```

As a sidenote, this prints _a lot_ of data so I'm not sure if we want this on by default.

Reply:
I do like that better.  If you don't mind I'm going to wait to make this change since I'm going to do a large refactor, and I'm pressed for time at the moment.

As for verbosity, that ship has sailed.  I think we'll address that problem using the technique Kumbi posted about though at some point.